api/spec/v1.yaml

```diff
@@ -1337,20 +1475,24 @@ components:
           example: 9e6a5837c6366d4a7e477c71ffe32d40915cdef7ef209792259e5ee70caf2705
         sender_0:
           type: string
-          description: The Oasis address of this transaction's 0th signer.
+          description: |
+            The Oasis address of this transaction's 0th signer.
+            Unlike Ethereum, Oasis natively supports multiple-signature transactions.
+            However, the great majority of transactions only have a single signer in practice.
+          # TODO: To access the other signers, ...
```

Comment:
join with the transaction_signers table :skull:

Reply:
Thanks. I left it as an internal note to ourselves, and documented for now that the API does not support retrieving other signers. It would be nice to have for sure, but let's wait for external demand first; we have plenty of TODOs.